companion/flashing.md

```diff
@@ -2,11 +2,11 @@
 
 ### Are Additional Drivers Required?
```

Comment:
Different style....  up to you... I obviously prefer my style but you have final say...

Reply:
You removed that part, not me :)